### PR TITLE
Giving up on pkg-up

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
Fixes https://github.com/googleapis/nodejs-firestore/issues/316

pkgUp seems to create more problems than it solves. This PR removes it and changes our source location to "dev/...", which allows us to find the package.json three levels up, whether we run from /build or from /dev.

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
This is kinda weird.  We don't have a `dev` folder in any of our other npm modules.  Why are we doing this?

**Comment by ofrobots** [comment]
@JustinBeckwith 
> This is kinda weird. We don't have a dev folder in any of our other npm modules. 

Actually we do. https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs has a top-level `ts/` to ensure that pre-compile relative paths are the same as post-compile relative paths.

File: dev/src/v1beta1/firestore_client.js [hunk 1]
```diff
@@ -18,9 +18,8 @@ const gapicConfig = require('./firestore_client_config');
 const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');
-const pkgUp = require('pkg-up');
 
-const VERSION = require(pkgUp.sync(__dirname)).version;
+const VERSION = require('../../../package').version;
```
Comment (ofrobots) [inline dev/src/v1beta1/firestore_client.js]: suggestion: I would recommend using the suffix `.json` even though it is not necessary.

Comment (schmidt-sebastian) [reply dev/src/v1beta1/firestore_client.js]: Done

**Pull request merged**